ci/travis/determine_tests_to_run.py

```diff
@@ -123,7 +124,9 @@ def get_commit_range():
         skip_prefix_list = ["doc/", "examples/", "dev/", "kubernetes/", "site/"]
 
         for changed_file in files:
-            if changed_file.startswith("python/ray/tune"):
+            if changed_file.startswith("python/ray/ml"):
+                RAY_CI_ML_AFFECTED = 1
```

Comment:
This should probably trigger tests for all dependent libraries, i.e. Serve, Train, Tune, RLLib

Reply:
I'll do this in a follow-up PR